tutorials/Basics/Idealized-Case-1_Tracking-of-a-Test-Blob-in-2D.ipynb

```diff
@@ -0,0 +1,3354 @@
R8TjwIXFhqQNgDeRVob/B+AVwFGdOHfTjXBJa5A+IbyG9EXDA8WuXYF/BPaXtGtELG5bLc3MbLg12rfT48V+ys81tS+euxlTPggxBHX1Os661+fvJwPyVorO5ZcBrwV2JjW+Vwf+QgpN6YhWfit8BtgBOAl4WkQ8OyKeDcwBvgTsCHy6bTU0MzMzs4EiUk943a3XdZT+pZhyez7wa+CjwGPAx4AXRcTGEfHOTp2/lXCUtwH/ERGHVyYWI0o/Lum5wNuBI6ZePTMzMzMbSP3fE/4V0ndH/4+09s0NMeXphRrXSk/4s8gsE1rhl0UeMzMzMxtWMcnWe78AlgDvAn4KfE/SIZI268bJW+kJXwBsUWf/84s8ZmZm3dXFecbzpyn5gn3KccJTncwsFyieV/oeeqircdIdmFtiWsV5N6EfZkCpJyJeL2kmsD0pFnwn4BvATEn3kOLB/zsi/l8nzt/K/+pLgQ9KekP1DkmvBz4AXDzVipmZmZnZ4FJE3a0fRMSyiLgqIo6OiB2A9Ulh1U8BBwLndOrcrfSEfwZ4A3CRpBuAW4r0FwHbAQ8DR7anemZmZmY2kPqjnT2pYua/V5FmR3kt8BLSnOHjwG87dd5WVsy8W9JLgeOB3UkVBXgc+AHwqYi4p31VNDMzM7NB0+/zhEs6khSG8nJgJmnClluBb5JCUS7v5FL2LS3WUzSy95Mk4GmkSj/UzRGlZmZmZtanGhl82ftW49HAn0ghJ5cBl0XEg906eUuN8AlFo/uhNtXFzMysMzqw2E/+NFNvVUx1YaCpmvog0A7o8ULcPR9YOaALkff7wExg015Gb0zaCJf0HFjR+73i58k4JMXMzMxsePV7OEqv26qN9ITfBYxLWjMilhY/N3JZR6dQLzMzMzMbZH3eCO+1Rhrhx5Iu4/Kqn83MzMzMsvq9J7zXJm2ER8TR9X42MzObNpqJvZ1i/Hh5FdrfcmlqAZ4BjT+e0PP47WYM+LWelOfrqKvp3yCS9pe0aZ39m0raf0q1MjMzM7OBpqi/DbtWPsafAfxDnf0vL/KYmZmZ2ZDSWP1t2LUyReFk32nNJK0wZGZmZmbDyr3ddbU6T3j2skqaDfwj8ECrFTIzMxsIU43n7VBMec5AxUkPkuke0z1FDjmpr6HfAJKOkjQmaYzUAP/exM+VG/AIsC/www7W2czMzMz6XUT9bcg12hP+W+BsUijK/sBVwJ1VeQJ4Arga+EGb6mdmZmZmA8g94fU11AiPiAuACwAkPRf4XET8dycrZmZmZmYDzI3wupqOCY+InTpRETMzMzObJgI02ViEIW+ktzJP+L6Szq6z/yxJb5latczMzKa5GO/MZrV8rXvC84TX18rQ7H+m/hSEY0WeSUl6lqSvSvqVpCckhaQdS/K+TtLVkhZJekjSt4rZWKrzrS3pZEkPFHmvk7RHt8o0MzMzM1JPd71tyLXSCN8KuKHO/huArRss6/nA20kDOktjzIuG+UXAn4HdgY8CewA/lWrmeJoL7Ad8hjRd4jxgrqRdO12mmZmZmSXuCa+vlXnC1yL1dpcJYJ0Gy7oyIp4OIGkvUiM45wTgZuBtEen7H0kPAJcAbwV+VKTtCuwC7BMRc4u0y4HNgZNIje5OlmlmZmZmAB2cn17SzsC7gFcAzwbmA78BjoqImzp24jZqpSf8T8Cr6ux/FXBPIwVNNH7rkbQJ8DLgnMr8EXEpcB/w5orsewMLKWZyKfIFcBawpaStO1WmmZlZX+hU/PMgb9YTGq+/TdH7gecA/w68CfhI8fO1krafculd0EpP+FzgE5IujYjTK3dIOpjUi3xiOypX2KZ4vTmz76aK/RN552Ua9zdW7u9QmStIWpApt9J6k+w3MzMzG2ydXZDnQxHxUGWCpEtIncUfY9UO1b7USiP8C8CewGmS/o20kE8ALybFgt8GHNem+gFsWLzOz+ybD7ykKu8fSvJVltWJMs3MzMys0Mm47+oGeJG2QNIfgWd17szt08o84Y9LeiVwPPA2Vg7CfBQ4FfhMRDzWviquPHWD6fVueaN5p1ImETG7Tv6JnnL3hpuZmdn01eXBl5KeRopQGIiV21vpCSciFgIflPQhYA5pOfu/FrHS7fZI8Zrrcd6AVXuzH6mTj4q8nSjTzMzMzEgNQ03SLFR6WW+yMN7JOjcBJAk4jTTe8UsNVbLHWhmYuUIkf42IhzrUAAe4pXjdJrNvW1aN674F2CozxeC2xevNFfnaXaaZmZmZTRifZGuvE4G9gPdHxK1tL70DWmqES1pH0pHFIjt/lPSKIn1Okb5luyoYEfcC1wH7VTaEi6lpNgHOq8g+F5hNmve70v7AbRExr1NlmpmZmVmhWLa+3laEqyyMiNn1tslOJenzwGHAhyPizI6+rzZqOhyliLf5FWme7NuL1zUAIuJhSQeQGq0fabC8iSXuX1a87iBpDvBkRPysSPs4af7uH0g6DdgY+CJwDXBuRXEXAZcDp0vakDRC9gDStIl7Vp26E2WamZmZGXR6dhQAJB0LfAo4PCJO7vgJ26iVmPDPAc8AXk6aD7x6dOoFwM5NlHdu1c9HF693A5sCRMRlknYDjgF+CjwOnE+64CsWDoqIKBb9Oa7YZpOmD9wnIi6sPEknyjQzMzOzpNOrYko6CjgCOCIi2jk9dle00gjfDTglIq4veoar3Qkc2GhhEaEG8/0c+HkD+R4DDi22rpdpZmZmZnS0J1zSYaSO2/8CflG1QM+SiLihYydvk1Ya4XNIYShlxoHVW6uOmZmZmU0HbVgVs56JsXq7FVulFdEU/ayVRvhfgOfV2b8dDS5bb2ZmZmbTVAd7wiNix44V3iWtzI5yEXCIpGdW75D0ctKsIRdMtWJmZmZmNrgmnR1lyLXSCD8GWA7cQFo1M4ADJP0AuBK4nzTLiJmZmZkNq4j625BruhEeEX8BtidN5XcwacGjdwH7kqb8e3VEeBVJMzMzs2HW3cV6Bk6ry9b/GdhT0rrAC0kN8dvd+DYzMzMz
```

Comment:
```suggestion
    "Another interesting property for real data are the lifetimes of our features. Tobac can also produce a histogram of this:"
```

Reply:
and the same: This comment refers to the image, right? Don't how to deal with it ...

Reply:
Same here, the only change I wanted to suggest was histgram -> histogram in l. 3297

Reply:
Thanks for careful reading! This is also corrected in the next commit.